Error looks real:
```
======================================================================
ERROR: corehq.apps.cleanup.tests.test_couch_to_sql_custom_data_fields:TestCouchToSQLCustomDataFieldsDefinition.test_migration
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/cleanup/tests/test_couch_to_sql_custom_data_fields.py", line 171, in test_migration
    call_command('populate_custom_data_fields')
  File "/vendor/lib/python3.6/site-packages/django/core/management/__init__.py", line 131, in call_command
    return command.execute(*args, **defaults)
  File "/vendor/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/mnt/commcare-hq-ro/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py", line 158, in handle
    self._verify_doc(doc)
  File "/mnt/commcare-hq-ro/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py", line 173, in _verify_doc
    exit(1)
  File "/usr/local/lib/python3.6/_sitebuiltins.py", line 26, in __call__
    raise SystemExit(code)
SystemExit: 1
```

--- a/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py
+++ b/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py
@@ -2,7 +2,7 @@ import logging
 import sys
 import traceback
 
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand, CommandError
 from django.core.management import call_command
 from django.db import transaction
 
@@ -141,6 +141,9 @@ class PopulateSQLCommand(BaseCommand):
     def handle(self, **options):
         verify_only = options.get("verify_only", False)
         skip_verify = options.get("skip_verify", False)
+
+        if verify_only and skip_verify:
+            raise CommandError("verify_only and skip_verify are mutually exclusive")
 
         self.doc_count = get_doc_count_by_type(self.couch_db(), self.couch_doc_type())
         self.diff_count = 0

--- a/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py
+++ b/corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py
@@ -154,10 +154,10 @@ class PopulateSQLCommand(BaseCommand):
         ))
         for doc in get_all_docs_with_doc_types(self.couch_db(), [self.couch_doc_type()]):
             self.doc_index += 1
-            if not skip_verify:
-                self._verify_doc(doc)
             if not verify_only:
                 self._migrate_doc(doc)
+            if not skip_verify:
+                self._verify_doc(doc)
 
         logger.info(f"Processed {self.doc_index} documents")
         if not skip_verify:

